src/responsive-layout.tsx

```diff
@@ -60,26 +15,19 @@ type Props = {
 };
 
 const ResponsiveLayout: React.FC<Props> = ({children, className, fullWidth, dataAttributes}) => {
-    const classes = useStyles();
     const {isMobile, isTablet} = useScreenSize();
 
-    const sideMargin = isMobile ? MOBILE_SIDE_MARGIN : isTablet ? TABLET_SIDE_MARGIN : null;
-
     const containerType = isMobile ? 'mobile-column' : isTablet ? 'tablet-column' : 'desktop-wide-column';
 
     return (
-        <ResponsiveLayoutMarginContext.Provider value={sideMargin}>
-            <ContainerTypeContext.Provider value={containerType}>
-                <div
-                    className={classnames(classes.container, className)}
-                    {...getPrefixedDataAttributes(dataAttributes)}
-                >
-                    <div className={classnames(classes.layout, {[classes.layoutFullWidth]: fullWidth})}>
-                        {children}
-                    </div>
-                </div>
-            </ContainerTypeContext.Provider>
-        </ResponsiveLayoutMarginContext.Provider>
+        <ContainerTypeContext.Provider value={containerType}>
```

Comment:
we talked about removing this context too, that it is feasible

Reply:
yes, I can give it a try

Reply:
I think I'll leave it to a different PR and migrate carousel component at the same time